src/main/ApplicationImpl.cpp

```diff
@@ -194,6 +195,16 @@ maybeRebuildLedger(Application& app, bool applyBuckets)
                 LOG_INFO(DEFAULT_LOG, "Dropping liquiditypools");
                 app.getLedgerTxnRoot().dropLiquidityPools();
                 break;
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+            case CONTRACT_DATA:
```

Comment:
missing schema upgrade/schema version bump in Database.cpp

Reply:
fixed.